content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,320 @@
+assigned directly on the sales order form.
 
-.. image:: serial_numbers/serial_numbers_07.png
-    :align: center
+Manage serial numbers on receipts
+---------------------------------
 
-Copy/pasting from an Excel file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Assigning serial numbers to **incoming** goods can be done directly from the purchase order (PO).
 
-To use copy/pasting, open the spreadsheet containing the serial numbers
-you received and copy the list. Then, past them in the *Lot/Serial
-Number Name* column of the wizard. Doing so, Odoo will automatically
-create the lines you need.
+To create a :abbr:`PO (purchase order)`, go to the :menuselection:`Purchase app --> Create`. Doing
+so reveals a new, blank request for quotation (RFQ) form.
 
-.. image:: serial_numbers/serial_numbers_08.png
-    :align: center
+On this :abbr:`RFQ (request for quotation)` form, fill out the necessary information, by adding a
+:guilabel:`Vendor`, and by adding the desired products to the :guilabel:`Product` lines, by
+clicking :guilabel:`Add a product`, under the :guilabel:`Products` tab.
 
-.. image:: serial_numbers/serial_numbers_09.png
-    :align: center
+Choose the desired quantity of the product to order, by changing the number in the
+:guilabel:`Quantity` column.
 
-.. image:: serial_numbers/serial_numbers_10.png
-    :align: center
+When the necessary configurations are complete, click :guilabel:`Confirm Order`. This will convert
+the :abbr:`RFQ (request for quotation)` to a purchase order.
 
-Operation types
-===============
+Then, click the :guilabel:`Receipt` smart button to be taken to the warehouse receipt form page for
+that specific :abbr:`PO (purchase order)`.
 
-You also have the possibility to define how you will manage lots for
-each operation type. To define it, go to :menuselection:`Configuration --> Operation Types` in the
-*Inventory* app.
+.. note::
+   Clicking :guilabel:`Validate` before assigning a serial number to the ordered product quantities
+   will cause a :guilabel:`User Error` pop-up to appear. The pop-up requires entry of a lot or
+   serial number for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be
+   validated without a serial number being assigned.
 
-For each type, you can decide if you allow the creation of new lot
-numbers or want to use existing ones. By default, the creation of new
-lots is only allowed at product reception.
+.. image:: serial_numbers/serial-numbers-user-error-popup.png
+   :align: center
+   :alt: User error popup prompting serial number entry.
 
-.. image:: serial_numbers/serial_numbers_11.png
-    :align: center
+From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
+vertical lines, located to the right of the :guilabel:`Unit of Measure` column in the
+:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```

Comment:
There are a couple ways you could format this, either with an icon or not both suggestions are below. I also changed the "four vertical lines" to "four horizontal lines" as hamburgers are horizontal 🍔 is this true on the UI? 
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a :guilabel:`≣ (hamburger)`
icon (four horizontal lines, located to the right of the :guilabel:`Unit of Measure` column in the
:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```
OR
```suggestion
From here, click the :guilabel:`Additional Options` menu, represented by a `hamburger` icon (four
vertical lines, located to the right of the :guilabel:`Unit of Measure` column in the
:guilabel:`Operations` tab). Clicking that icon reveals a :guilabel:`Detailed Operations` pop-up.
```